kokoro/scripts/build/build_package.ps1

```diff
@@ -47,10 +47,24 @@ PACKAGE_VERSION,$env:PKG_VERSION
   Out-File -FilePath "$env:KOKORO_ARTIFACTS_DIR/custom_sponge_config.csv" -Encoding ascii
 
 Invoke-Program git submodule update --init
-Invoke-Program docker build -t $tag -f './Dockerfile.windows' .
+$artifact_registry="us-docker.pkg.dev"
```

Comment:
go/powershell-style says to use single quotes unless we are doing any fancy string interpolation, which we aren't here.